man/man5/zpool-features.5

```diff
@@ -383,6 +383,13 @@ READ\-ONLY COMPATIBLE	no
 DEPENDENCIES	enabled_txg
 .TE
 
+This feature has/had bugs, the result of which is that, if you do a
+\fBzfs send -i\fR (or \fB-R\fR, since it uses \fB-i\fR) from an affected
+dataset, the receiver will not see any checksum or other errors, but will not
+match the source.  Its use by \fBzfs send -i\fR has been disabled by default.
```

Comment:
but the resulting destination snapshot will not match....

Reply:
Fixed, in this PR and the wiki.